test/functional/test-amp-img.js

```diff
@@ -374,4 +377,80 @@ describe('amp-img', () => {
     impl.unlayoutCallback();
   });
 
+  describe('blurred image placeholder', () => {
+    let el;
+    let img;
+    let impl;
+
+    beforeEach(() => {
+      getImgWithBlur(true, true);
+      toggleExperiment(impl.win, 'blurry-placeholder', true, true);
+    });
+
+    afterEach(() => {
+      impl.unlayoutCallback();
+    });
+
+    /**
+     * Creates an amp-img with an image child that could potentially be a
+     * blurry placeholder.
+     * @param {boolean} addPlaceholder Whether the child should have a
+     *     placeholder attribute.
+     * @param {boolean} addBlurClass Whether the child should have the
+     *     class that allows it to be a blurred placeholder.
+     */
+    function getImgWithBlur(addPlaceholder, addBlurClass) {
```

Comment:
This test code would be easier to follow if this helper was idempotent. I.e. change it to return the `AmpImg` instead of modifying existing variables. 

```js
it('my test', () => {
  const impl = getImgWithBlur(...);
  const el = impl.element;
});
```